servlet/src/main/java/org/togglz/servlet/activation/UrlParameterActivationStrategy.java

```diff
@@ -0,0 +1,117 @@
+package org.togglz.servlet.activation;
+
+import org.togglz.core.activation.ParameterBuilder;
+import org.togglz.core.repository.FeatureState;
+import org.togglz.core.spi.ActivationStrategy;
+import org.togglz.core.user.FeatureUser;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Pattern;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URIBuilder;
+import org.togglz.core.activation.Parameter;
+import org.togglz.core.util.Strings;
+import org.togglz.servlet.util.HttpServletRequestHolder;
+import javax.servlet.http.HttpServletRequest;
+
+public class UrlParameterActivationStrategy implements ActivationStrategy {
```

Comment:
Could you perhaps apply the Toggz code format here?
See: https://github.com/togglz/togglz/blob/master/eclipse-code-format.xml